lib/jwt/jwt.go

```diff
@@ -32,7 +32,6 @@ import (
 	"github.com/jonboulle/clockwork"
 	"gopkg.in/square/go-jose.v2"
 	"gopkg.in/square/go-jose.v2/cryptosigner"
-	"gopkg.in/square/go-jose.v2/jwt"
 	josejwt "gopkg.in/square/go-jose.v2/jwt"
```

Comment:
Not to be a pain, but if we're going to remove one of the duplicates, I would suggest the aliased one.

We don't need the `josejwt` alias because there is not another package `jwt` in scope. Let's just import the package using it's standard name.

Reply:
I agree. If @russjones is ok with it I'll do the changes and update the pull request.

Reply:
@fredcarle Yes, can you please apply @zmb3's suggestion when you get a chance? Then we can merge this. Thanks!

Reply:
@r0mant done :)